src/components/organisms/SettingsManager/Settings.tsx

```diff
@@ -350,7 +360,11 @@ export const Settings = ({
           onChange={onChangeScanExcludes}
           tooltip={AddExclusionPatternTooltip}
           isSettingsOpened={isSettingsOpened}
-          type="excludes"
+          showApplyButton={appConfig.isScanExcludesUpdated === 'outdated'}
+          onApplyClick={() => {
+            dispatch(setScanExcludesStatus('applied'));
```

Comment:
does this have to be a separate reducer? or could the setRootFolder reducer set this property? (would apply above also)

Reply:
we should set includes and excludes in the `setRootFolder`, are we ok with this?

Reply:
Thanks for the suggestions, it looks a lot more cleaner now